src/components/updatemenus/scrollbox.js

```diff
@@ -246,6 +246,8 @@ ScrollBox.prototype.enable = function enable() {
                 .on('.drag', null)
                 .call(onBarDrag);
         }
+
+        this.container.on('wheel', this._onBoxWheel.bind(this));
```

Comment:
Currently, the wheel handler isn't called when one tries to scroll with the cursor over the gap between the buttons (you can try increasing the `gapButton` constant) to make this problem more obvious).

I think the easiest solution would be to make `Scrollbox` add a background `<rect>` as big as the clipped `<rect>` in the container and attach the `wheel` handler on it.